modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -1867,11 +1984,83 @@ public BaselineAutoAdjustStatus baselineAutoAdjustStatus(){
     }
 
     /**
-     * @param activate Activate.
-     * @return Activate flag string.
+     * @param state Cluster state.
+     * @return Cluster state string representation.
+     */
+    private static String prettyStr(ClusterState state) {
+        switch (state) {
+            case ACTIVE:
+                return "activate cluster";
+
+            case INACTIVE:
+                return "deactivate cluster";
```

Comment:
you need noun not verb here, "inactive cluster" ?

Reply:
Sorry, but we already have these messages in logs. See this method in a master branch. I'd prefer to don't change messages in the log.